src/analysis/classic/hunter/survival/CONFIG.tsx

```diff
@@ -6,23 +7,24 @@ import CHANGELOG from './CHANGELOG';
 
 const config: Config = {
   // The people that have contributed to this spec recently. People don't have to sign up to be long-time maintainers to be included in this list. If someone built a large part of the spec or contributed something recently to that spec, they can be added to the contributors list. If someone goes MIA, they may be removed after major changes or during a new expansion.
-  contributors: [],
-  expansion: Expansion.TheBurningCrusade,
+  contributors: [bdfreeman1421],
+  expansion: Expansion.WrathOfTheLichKing,
   // The WoW client patch this spec was last updated.
-  patchCompatibility: '2.5.1',
+  patchCompatibility: '3.4.0',
   isPartial: true,
   // Explain the status of this spec's analysis here. Try to mention how complete it is, and perhaps show links to places users can learn more.
   // If this spec's analysis does not show a complete picture please mention this in the `<Warning>` component.
   description: <>Classic WotLK support is still a Work in Progress.</>,
   pages: {
     overview: {
-      hideChecklist: true,
+      hideChecklist: false,
       text: <>Classic WotLK support is still a Work in Progress.</>,
       type: 'info',
     },
   },
   // A recent example report to see interesting parts of the spec. Will be shown on the homepage.
   exampleReport: '/report/8AaZGQvkzJdDYfMR/11-Normal+Magtheridon+-+Kill+(7:42)/Badmera',
```

Comment:
Update to a more recent example report. You can search top dps survival hunters on WCLs:
https://classic.warcraftlogs.com/zone/rankings/1017#metric=dps&partition=2&class=Hunter&spec=Survival

Reply:
I grabbed the top hunter from my realm. Not sure of the protocol but figured that was as good a strategy as any. Took me a while to find where it was displayed on the UI/interface to check the link was correct.

Reply:
Alot more messages in console with a hunter with different play style. I'll have to sample a few more. The EXPLOSIVE_TRAP gcd seems to be overlapping with serveral other spells so not sure whats going on in the gcd overlap tracker with that spell.